app/frontend/startup.js

```diff
@@ -312,7 +312,8 @@ class Startup {
     $('#loading-subtitle').hide();
 
     // Show main content
-    $('#main-content').show();
+    document.getElementById('main-content').style.display = 'block';
+    document.getElementById('panel-buttons').style.display = 'block';
```

Comment:
@tobias-klein `panel-buttons` are inside `main-content`. Is there a problem displaying it?

Reply:
There was not a problem displaying them, but rather a problem that they were displayed too early. I noticed this when I tested it on Android. They were already visible at the point when the permissions message was shown at initial startup, but at that point nothing is supposed to be visible aside from that dialog with the permissions message. This adjustment (`display: none`) by default and explicitly showing it solved the issue. If we find a better solution later, we can still change this.

Reply:
I guess my question was why should `panel-buttons` also be hidden and reshown as it is a descendant of `main-content` which the default hidden by default.

Reply:
![Screenshot (Dec 5, 2021 10_15_39)](https://user-images.githubusercontent.com/17739895/144741218-2832e742-2604-46ca-ad39-13d929f01226.png)

This is how it looked on Android before my latest change. You see that there are some panel buttons shown - based on the element in index.html (even though not initialized yet).

The reason is that the panel-buttons are (contrary to other components) in the DOM (index.html) right from the beginning and were not hidden by default. Looking at the startup sequence, other components are only loaded after the Android permissions have been handled.

What happens before the Android permissions box is shown is that this function is called:
```JavaScript
  hideGlobalLoadingIndicator() {
    var loadingIndicator = $('#startup-loading-indicator');
    loadingIndicator.hide();
    $('#main-content').show();
  }
```

At that point the panel-buttons were then becoming visible, even though they shouldn't. Other components didn't show the same issue, because they weren't even loaded in the DOM at that point. Note that after the Android permissions are acknowleged by the user, the global loading indicator is shown again (and main-content hidden), consider this piece of code from `cordova_platform.js`:

```JavaScript
  onPermissionGranted() {
    console.log("Permission to access storage has been GRANTED!");
    this.getPermissionBox().dialog('close');
    uiHelper.showGlobalLoadingIndicator();
    this.initPersistenceAndStart();
  }
```

As a fix I made the panel-buttons invisible by default (`display: none` in main.css) and am activating them now during the startup sequence. To make it easier to understand I moved the activation to the function `loadHTML` (in startup.js) with some comment.

Reply:
@tobias-klein thank you for detailed explanation! Now I understand the problem.
I know you already merge this, so I have a bit different idea in another PR.